CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 6.22.0 (December 19, 2022). Tested on Artifactory 7.47.12
+
+IMPROVEMENTS:
+
+* Added new datasource: datasource/datasource_artifactory_group.go
+  PR: [#PR](https://github.com/jfrog/terraform-provider-artifactory/pull/607)
+  Issue: [#548](https://github.com/jfrog/terraform-provider-artifactory/issues/548)
```

Comment:
Please follow the examples in this file, where `Issue` comes first.

Reply:
I did, the first example I could find is version 6.9.0 and 6.7.2 where PR comes first.